### PR TITLE
test: verify_clean_boot enhancements

### PR DESCRIPTION
**Pull request by TheRealFalcon**
## Proposed Commit Message

```
test: verify_clean_boot enhancements
```

## Merge type

- [x] Squash merge using "Proposed Commit Message"
- [ ] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

File: tests/integration_tests/util.py [hunk 1]
```diff
@@ -352,29 +355,13 @@ def _verify_clean_boot(
...
         if CURRENT_RELEASE < NOBLE:
-            if os.environ.get("ON_JENKINS"):
-                # We are on Jenkins testing against full packages
-                # Old releases return 0 for backwards compatibility
-                rc = 0
-            elif os.environ.get("GITHUB_ACTION"):
-                # On GH actions, integrations tests run on LXD without quilt
-                # patches
-                rc = 2
+            rc = 0
```
Comment (TheRealFalcon) [inline tests/integration_tests/util.py:364]: Is this a reasonable assumption now?

Comment (blackboxsw) [reply tests/integration_tests/util.py:364]: Now that 0f77b3c13a2f44c224c5dea18d27bbfa4898e39a landed moving us to oracular which is RC = 2 this should be reasonable and no need to check GITHUB_ACTION.

**Review by blackboxsw — APPROVED** [review]

**Comment by holmanb** [comment]
```
        # Look at the commit history before making changes here.
        # Behavior on Jenkins, GH CI, and local runs must be considered
        # for all series.
```
XD

Yeah, fair enough.

**Review by holmanb — APPROVED** [review]

**Comment by holmanb** [comment]
I think that we need to set `CLOUD_INIT_OS_IMAGE=oracular` so that we actually launch oracular 
images.

**Pull request merged**